torrentfile/commands.py

```diff
@@ -419,10 +419,11 @@ def magnet(metafile: str, version: int = 0) -> str:
     magnet += trackers if trackers != "&tr=" else ""
 
     if "url-list" in meta:
-        web_seed = [
-            "&ws=" + quote_plus(url) for urllist in meta["url-list"]
-            for url in urllist
+        web_sources = [
+            "&ws=" + quote_plus(urllist) for urllist in meta["url-list"]
             ]
+    web_seed = "".join(web_sources)
```

Comment:
This line is still causing an issue.  When trying to create a magnet link that doesn't contain webseeds then the if block will not be entered and therefore `web_sources` is never created so this line triggers a failure.